okhttp/build.gradle.kts

```diff
@@ -9,12 +13,27 @@ plugins {
   id("binary-compatibility-validator")
 }
 
-// Build & use okhttp3/internal/-InternalVersion.kt
+fun ByteArray.toByteStringExpression(): String {
+  return "\"${Base64.getEncoder().encodeToString(this@toByteStringExpression)}\".decodeBase64()!!"
```

Comment:
Seems like `byteArrayOf` would save even more space since it would be stored raw instead of as a string that is then encoded to bytes for the classfile.

Reply:
Yep, I'll try that. I was hitting limits of Kotlin? compiler before, with the source file like "...12, 15, -34, -56, 12..."

But let me try again

Reply:
Source back to 150KB, Kotlin compiler is noticeably struggling with the build. When the compile completes I'll report on the class file.

Reply:
I've given up on that for now.

Reply:
Probably worth filing a Kotlin issue if it's not fast. Although this project is stuck on old Kotlin with the old compiler, I believe.

Could also just generate package-private Java with a single static field.

Reply:
```
C:\Users\yuri\IdeaProjects\okhttp\okhttp\build\generated\sources\kotlinTemplates\okhttp3\internal\publicsuffix\PublicSuffixDatabaseGZ.java:29: error: code too large
  public static final byte[] compressedBytes = new byte[] {31,-117,8,0,0,0,0,0,0,0,-116,-67,-33,-114,-60,-88,-46,39,-72,123,91,-113,50,82,-90,78,119,-97,-17,-100,-39,-57
```

Reply:
If I split to 10 methods, it works but they don't get optimised as a single array for some reason?

If I split to 10 constants, still the same error.

Reply:
Sure. I'll look tomorrow